Tools/TestWebKitAPI/Configurations/Base.xcconfig

```diff
@@ -37,7 +37,7 @@ CLANG_WARN_CXX0X_EXTENSIONS = NO;
 ALTERNATE_HEADER_SEARCH_PATHS = $(ALTERNATE_HEADER_SEARCH_PATHS_$(SDK_VARIANT));
 ALTERNATE_HEADER_SEARCH_PATHS_iosmac = $(BUILT_PRODUCTS_DIR)$(WK_ALTERNATE_FRAMEWORKS_DIR)/usr/local/include
 
-PROJECT_HEADER_SEARCH_PATHS = $(SRCROOT)/../../Source/WebKit/Platform/cocoa $(PROJECT_HEADER_SEARCH_PATHS_$(WK_COCOA_TOUCH));
+PROJECT_HEADER_SEARCH_PATHS = $(SRCROOT)/../../Source/WebKit/Platform/cocoa $(SRCROOT)/../../Source/WebGPU $(PROJECT_HEADER_SEARCH_PATHS_$(WK_COCOA_TOUCH));
```

Comment:
Are there specific imports that this fixes? Asking because this PR changes many WGSL quote imports to library imports, so I want to make sure this is still needed.

Reply:
I was having troubles getting another xcodeproj to access the WGSL headers because WebGPU.xcodeproj uses header maps. I eventually promoted the internal headers to private headers so they are installed in $(BUILT_PRODUCTS_DIR) and TestWebKitAPI can use flat `#include <WGSL/header.h>`.

Please let me know if there is a better way to do this!